packages/adapter-node/index.d.ts

```diff
@@ -1,10 +1,6 @@
 import { Adapter } from '@sveltejs/kit';
 import './ambient.js';
 
-declare global {
-	const ENV_PREFIX: string;
```

Comment:
this was previously in the exposed types, but I guess is no longer exposed since it's been moved to `ambient.d.ts`? would users need this type or would this be a breaking change?

Reply:
Hmm, not sure, feels like this shouldn't have been exposed in the first place. But if anyone is relying on it then yes it could be a breaking change since ENV_PREFIX isn't replaced everywhere anymore, it only replaces `import.meta.ENV_PREFIX` inside a specific module.

Reply:
I think you may need the `ENV_PREFIX` if you're building a [custom server](https://kit.svelte.dev/docs/adapter-node#custom-server)? E.g. in the example in the docs it just hardcodes 3000 for the port, but if you wanted to get it from `process.env[ENV_PREFIX + 'PORT']`, then you're probably need some way to get the ENV_PREFIX. I don't know if there's another way of getting it perhaps

@Conduitry I think you may have requested this feature originally? Do you have any thoughts on this?

Reply:
Sorry, I had missed this question before.

I think we don't want to try to expose `ENV_PREFIX` to the user's custom server code. I'm not sure how that would work, and I don't think we are doing that now. The user's entrypoint is essentially going to be unbundled JS that imports the `handler.js` module that the adapter generates for them. We give people `ENV_PREFIX` so they can change the behavior of code they don't have direct access to - which is the handler function and (for people using the default server) the actual instantiation of the Node HTTP server.

If you're replacing the entrypoint `index.js` file, it should be your responsibility to make it behave how you want. That might mean listening on specific interfaces depending on certain environment variables, but it could also mean doing something entirely different.

If we feel like it would be safer, we could say that this is a breaking change. I don't think we have to, though, because I think the custom entrypoint already doesn't have access to the env prefix.

Reply:
IIUC As it's implemented now, any usage of ENV_PREFIX in the resulting bundle is replaced whether it's in the adapter provided files or any from the user.

Reply:
The entrypoint for the custom server isn't bundled though, is it? I thought that was just a file the user wrote that then imports the `handler.js` that we do generate with the adapter.

Reply:
Ahh, you're right. I misread it thinking it was replacing the global everywhere but it only does so in the files that it copies, i.e. the ones in the adapter-node/files directory. Shouldn't be a breaking change.